salt/cloud/clouds/proxmox.py

```diff
@@ -925,6 +927,7 @@ def create_node(vm_, newid):
     if "clone" in vm_ and vm_["clone"] is True and vm_["technology"] == "qemu":
         postParams = {}
         postParams["newid"] = newnode["vmid"]
+        postParams["name"] = vm_["name"]
```

Comment:
Also note that this also is a change. Now it will always set the name of the VM in the Proxmox GUI to the name it is given from the cloud profile. Without this the clone would have the name of the template prefixed by `Copy-of-VM-`. The name could still be overruled by setting `clone_name` setting from cloud profile.